source/features/comment-fields-keyboard-shortcuts.tsx

```diff
@@ -12,10 +12,7 @@ function eventHandler(event: delegate.Event<KeyboardEvent, HTMLTextAreaElement>)
 
 	if (event.key === 'Escape') {
 		// Cancel buttons have different classes for inline comments and editable comments
-		const cancelButton = select(`
-			button.js-hide-inline-comment-form,
-			button.js-comment-cancel-button
-		`, field.form!);
+		const cancelButton = select('button:is(.js-hide-inline-comment-form, .js-comment-cancel-button)', field.form!);
```

Comment:
This turns 2 clear lines in a long multiline. It’s one of those times when :is() is not useful